package.json

```diff
@@ -106,6 +107,8 @@
     "start:service-worker": "yarn build && yarn serve -s build",
     "start:default": "craco start",
     "build": "craco build",
+    "ipfs:build": "cross-env PUBLIC_URL=\".\" react-scripts build",
+    "ipfs:publish": "ipfs-deploy -p infura -O dist ./build",
```

Comment:
what is `dist`? we use `build` folder so is this a typo or sth part of `ipfs-deploy`?